src/main/java/hudson/plugins/sidebar_link/SidebarLinkPlugin.java

```diff
@@ -178,4 +183,9 @@ public boolean isAcceptedIconName(String iconName) {
         // accepts .gif extension as a compatibility with older version of the plugin
         return iconName.matches(("^[\\w-]+(\\.gif)?$"));
     }
+
+    private boolean isAcceptedSymbol(String iconName) {
+        return iconName.matches("^symbol-[-\\w]+(\\s+plugin-[-\\w]+)?\\s*$");
```

Comment:
accepted for what?
what this regexp refers to?
would be nice to have some docs for future maintenance

Reply:
See https://weekly.ci.jenkins.io/design-library/Symbols/ how to use symbols.
You can already use them now when you ignore the Form validation error.
+1 for updating the help. It refers to the 24x24 images which are anyway outdated.

Reply:
I'm closing this PR. Don't have time to update it. Feel free to take it and update doc and forma validation the way you wish

Configuration of symbol icon works also great by JCasC config